SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DonationsMoreDetailsController.swift

```diff
@@ -24,8 +24,45 @@
 
 import UIKit
 
+protocol CommunitySupportEntity {
```

Comment:
Im not sure this protocol and extensions should be inside the details controller. What is the purpose of this?

Reply:
this controller supports working with data that conforms to this protocol, so fits in this class I think.
naming-wise it's not super clear what to do with it, because the donation ui related classes will be renamed too, but don't know to what yet as it's a follow-up task.